NewHaven/src/Player.cpp

```diff
@@ -2,14 +2,14 @@
 #include "GBMap.h"
 Player::Player(){
     village = new VGMap();
-    resource_score = new ResourceScore();
+    resource_score = new ResourceTracker();
     vb_score = new ScoreCalculator();
     my_hand = new Hand();
 }
 
 Player::Player(const Player &player) {
     village = new VGMap(*player.village);
-    resource_score = new ResourceScore(*player.resource_score);
+    resource_score = new ResourceTracker(*player.resource_score);
     vb_score = new ScoreCalculator(*player.vb_score);
     my_hand = new Hand(*player.my_hand);
 }
```

Comment:
The resource tracker is initialized, but there's still unnecessary copying, use list initialization.
ex: `resource_score{new ResourceTracker(*player.resource_score),`
Also why did we switch to snake case variable naming?

Reply:
I've always use snake case for variable names. is that an issue?